targets/TARGET_NORDIC/TARGET_NRF5/TARGET_MCU_NRF51822_UNIFIED/sdk/softdevice/s130/headers/ble.h

```diff
@@ -0,0 +1 @@
+#include "nrf_ble.h"
```

Comment:
If softdevice header files are not prefixed then we will encounter issues on windows machine between `BLE.h` and `ble.h` like it happened in the past. `nrf_ble.h` has always been named `ble.h` in the Nordic SDK, it was always renamed during integration.